Gemfile.lock

```diff
@@ -1038,8 +1072,9 @@ GEM
     zeitwerk (2.6.13)
 
 PLATFORMS
+  arm64-darwin-22
```

Comment:
https://github.com/department-of-veterans-affairs/vets-api/pull/15143#issuecomment-1984711913

bundle lock --add-platform ruby
bundle lock --remove-platform arm64-darwin-22
bundle install --redownload